src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal.tsx

```diff
@@ -0,0 +1,191 @@
+    if (!relayEnvironment) {
+      return null
+    }
+
+    try {
+      await submitMutation({
+        variables: { input: { name: values.name } },
+        rejectIf: response => {
+          const result = response.createCollection?.responseOrError
+          return result?.__typename === "CreateCollectionFailure"
+            ? result.mutationError
+            : false
+        },
+      })
+
+      onComplete()
+    } catch (error) {
+      logger.error(error)
+
+      setFormFieldError(
+        error.message ?? t("collectorSaves.createNewListModal.genericError")
+      )
+    }
+  }
+
+  return (
+    <Formik<CreateNewListValues>
+      initialValues={{ name: "" }}
+      onSubmit={handleSubmit}
+    >
+      {({ values, isSubmitting, handleSubmit, handleChange, handleBlur }) => {
+        const isCreateButtonDisabled = !values.name
+
+        return (
+          <ModalDialog
+            width={["100%", 713]}
+            onClose={onClose}
+            title={t("collectorSaves.createNewListModal.title")}
+            data-testid="CreateNewList"
+            footer={
+              <>
+                {/* Desktop view */}
+                <Media greaterThanOrEqual="sm">
+                  <Flex justifyContent="flex-end">
+                    <Button
+                      disabled={isCreateButtonDisabled}
+                      loading={isSubmitting}
+                      onClick={() => handleSubmit()}
+                    >
+                      {t("collectorSaves.createNewListModal.createList")}
+                    </Button>
+                  </Flex>
+                </Media>
+
+                {/* Mobile view */}
+                <Media lessThan="sm">
+                  <Button
+                    disabled={isCreateButtonDisabled}
+                    loading={isSubmitting}
+                    onClick={() => handleSubmit()}
+                    width="100%"
+                  >
+                    {t("collectorSaves.createNewListModal.createList")}
+                  </Button>
+
+                  <Spacer y={1} />
+
+                  <Button
+                    variant="secondaryBlack"
+                    width="100%"
+                    onClick={onClose}
+                  >
+                    {t("collectorSaves.createNewListModal.back")}
+                  </Button>
+                </Media>
+              </>
+            }
+          >
+            <LabeledInput
+              title={t("collectorSaves.createNewListModal.nameLabel")}
+              name="name"
+              placeholder={t(
+                "collectorSaves.createNewListModal.namePlaceholder"
+              )}
+              value={values.name}
+              onChange={e => {
+                handleChange("name")(e)
+                setRemainingCharacters(
+                  Math.max(maxNameLength - e.target.value.length, 0)
+                )
+                setFormFieldError("")
+              }}
+              onBlur={handleBlur("name")}
```

Comment:
i think we can pass just `handleBlur` (without field name as arg). Something like this `onBlur={handleBlur}`